### PR TITLE
Use FTCreateParams for ftCreate method

### PR DESCRIPTION
**Pull request by sazzad16**
To do:
- [x] Access 'name' of SchemaField

### CODE REVIEW

**Comment by bsbodden** [comment]
@sazzad16 other than missing the SchemaField#name, the changes look good and consistent in Jedis!

File: redis-om-spring/pom.xml [hunk 1]
```diff
@@ -61,7 +61,7 @@
     <java.version>17</java.version>
     <spring.version>3.1.3</spring.version>
     <sdr.version>3.1.3</sdr.version>
-    <jedis.version>4.4.3</jedis.version>
+    <jedis.version>4.5.0-SNAPSHOT</jedis.version>
```
Comment (sazzad16) [inline redis-om-spring/pom.xml:64]:
```suggestion
    <jedis.version>4.4.6</jedis.version>
```

File: redis-om-spring/pom.xml [hunk 2]
```diff
@@ -96,6 +96,9 @@
     <repository>
       <id>snapshots-repo</id>
       <url>https://s01.oss.sonatype.org/content/repositories/snapshots</url>
+    </repository><repository>
+      <id>jedis-snapshots-repo</id>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
```
Comment (sazzad16) [inline redis-om-spring/pom.xml:101]:
```suggestion
    </repository>
```

**Review by sazzad16 — COMMENTED** [review]
Will move to GA release once Jedis has a new one.
Requesting not to merge until then.

File: redis-om-spring/pom.xml (hunk 1 above)
Comment (bsbodden) [reply redis-om-spring/pom.xml:64]: @sazzad16  The PR looks great, the only problem is that at runtime there is no way for us to override the Jedis version coming from Spring Data Redis - so we'll have to wait for them to up the version to merge - or perhaps we should create a Jedis 4.5 branch?

**Review by bsbodden — DISMISSED** [review]
This is perfect, thanks @sazzad16  now we just need to wait for Spring to up the Jedis version - I'll merge this when they do!

**Comment by bsbodden** [comment]
Cherry-picked on separate PR

**Pull request closed without merging**